### PR TITLE
Feature/78 toast

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**

### CODE REVIEW

**Comment by ElishaSamPeterPrabhu** [comment]
pc Screenshots toast
pc1
pc2

**Review by Sofiya-kumar — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Images/danger.png
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Images/danger.png:1]: Svg icons are unable to load hence, we are converting them to pngs amd using them

File: Trimble.Modus.Components/Themes/Colours.xaml [hunk 1]
```diff
@@ -25,13 +26,13 @@
...
     
     <!--ToastColors-->
-    <Color x:Key="ToastBlue">#DCEDF9</Color>
-    <Color x:Key="ToastWhite">#FFFFFF</Color>
-    <Color x:Key="ToastBlack">#252A2E</Color>
-    <Color x:Key="ToastGrey">#CBCDD6</Color>
-    <Color x:Key="ToastRed">#FBD4D7</Color>
-    <Color x:Key="ToastYellow">#FFF5E4</Color>
-    <Color x:Key="ToastGreen">#E0ECCF</Color>
+    <Color x:Key="Primary">#DCEDF9</Color>
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Themes/Colours.xaml:29]: Colors are duplicated but the key we are using will help us to identify the purpose of using that color and match them easily with the themes mentioned in figma ,hence we have separate colors keys for toast.

**Comment by ElishaSamPeterPrabhu** [comment]
> @ElishaSamPeterPrabhu I have made some alignment fixes for iOS and also moved the close and action button to the xaml. Please run and check before merging. Please check Sofia's comment too

@rthanga1 Modified some padding and margin to fix alignment .Can you run it in iOS before merging?

File: Trimble.Modus.Components/Themes/Colours.xaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (coliff) [reply Trimble.Modus.Components/Themes/Colours.xaml:29]: Related - We should rename `colours.xaml` to `colors.xaml` for consistency. We should always use US English where possible.

**Pull request merged**